data/mods/Craft_Gunpowder/cgp_recipes.json

```diff
@@ -24,40 +24,6 @@
   ]
 },{
   "type" : "recipe",
-  "result": "8mm_bootleg",
-  "category": "CC_AMMO",
-  "subcategory": "CSC_AMMO_BULLETS",
-  "skill_used": "fabrication",
-  "skills_required": [ "cooking", 2 ],
-  "difficulty": 7,
-  "time": 45000,
-  "book_learn": [[ "recipe_caseless", 5 ] , ["recipe_bullets" , 9 ] , ["manual_pistol", 10 ] , ["manual_rifle", 10 ]],
-  "qualities":[ {"id":"CHEM","level":2} ],
-  "tools": [
-    [
-      [ "press", 50 ],
-      [ "hotplate", 50 ],
-      [ "toolset", 50 ],
-      [ "fire", -1 ]
-    ],
-    [ [ "press", -1 ] ]
-  ],
-  "components": [
-    [ [ "acid", 1 ] ],
-    [ [ "plastic_chunk", 5 ] ],
-    [ [ "oxy_powder", 100 ] ],
-    [ [ "incendiary", 100 ] ],
-    [
-      [ "lead", 75 ],
-      [ "gold_small", 75 ],
-      [ "silver_small", 75 ],
-      [ "tin", 75 ],
-      [ "bismuth", 75 ],
-      [ "solder_wire", 75 ]
-    ]
-  ]
-},{
-  "type" : "recipe",
```

Comment:
So, uh. You removed bootleg jsp rounds but left the recipe for said rounds, and then removed the recipe for the standard bootleg rounds. What happens when you attempt to craft bootleg 8mm in this case?